core/common.go

```diff
@@ -12,3 +16,10 @@ func EmptyChannel(ch chan bool) int {
 		}
 	}
 }
+
+// UniqueIdentifier returns a unique string identifier
```

Comment:
might add ...of 32 bytes long
Since this will contain unprintable elements, I guess (for the sake of the re usability) to change the return type in []byte

Reply:
done